### PR TITLE
Update to Guile 2.24

### PR DESCRIPTION
**Pull request by JL2210**
Works on #3
Please test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,16 @@ licence and version as Guile)
...
 Kai-Martin Knaak
-
+James Larrowe
 
 How to use
 -------------------------------------------------------------------------------
 
 By calling:
 
-    $ ./build-commit v2.0.9
```
Comment (Madsy) [inline README.md:34]: You removed the example of how to build a specific commit. It's important that people know that the argument to ./build-commit is an actual git hash or git tag. Basically don't make any new changes to the README.md file, except for:

* adding your name to the contributor list,
* bumping the latest working guile version whenever the guile version is mentioned in the examples
* add all the quirks and workarounds regarding the new version in the bugs list at the very end

Comment (JL2210) [reply README.md:34]: It's actually not. Now that this uses release tarballs, you can no longer build git commits. I haven't fully tested all the (possibly new) quirks and workarounds yet, as I don't have my mingw-w64 toolchain built yet. I've pre-downloaded all the minor revisions of guile 2.0+ with the latest patchlevels to show that those are the most stable/supported ones. Once I can test, I will definitely make some new changes.

Comment (JL2210) [reply README.md:34]: In case you haven't noticed, I've renamed `build-commit` to `build-release` and removed all of the Git-specific references.

Comment (JL2210) [reply README.md:34]: Have you tried this? Does it work?

**Pull request closed without merging**